pkg/config/config.go

```diff
@@ -73,6 +73,7 @@ type CSVConfig struct {
 }
 
 type File struct {
+	Paths        []string
```

Comment:
I think this field is not necessary. you can also use the following `Path` to represent data directory and check whether the path is directory in file reader.